NeoML/include/NeoML/Dnn/DnnBlob.h

```diff
@@ -136,6 +136,11 @@ class NEOML_API CDnnBlob : public IObject {
 	// Copies the contents from another blob
 	void CopyFrom(const CDnnBlob* other);
 
+	// Transfers CDnnBlob data from other thread owner to this thread.
+	// CDnnBlob can be transfered, if at least one pointer to it is still exist, and on other side
```

Comment:
You can't call method of an objects if there is no pointers to it. So this part of the comment is excessive. And mb the users will change this association without stopping of allocated thread. I think it's better to rewrite it into smth like:

By default memory underneath each blob is associated with the thread on which the allocation of the blobhas occurred.
This method switches this association to the calling thread.

Reply:
fixed